rust-app/src/implementation.rs

```diff
@@ -92,14 +92,48 @@ const AMOUNT_ACTION: Action<AmountType<JsonStringAccumulate<64>, JsonStringAccum
         });
 */
 
-type SendMessageAction = impl JsonInterp<SendValueSchema, State: Debug>;
+type SendMessageAction = impl JsonInterp<SendValueSchema, State: Debug, Returning = ()>;
 const SEND_MESSAGE_ACTION: SendMessageAction = Preaction(
-    || scroller("Send", |w| Ok(write!(w, "Transaction")?)),
-    SendValueInterp {
-        field_amount: VALUE_ACTION,
-        field_from_address: show_address::<"Transfer from">(), // FROM_ADDRESS_ACTION,
-        field_to_address: show_address::<"Transfer To">(),
-    },
+    || scroller("Transfer", |w| Ok(write!(w, "POKT")?)),
+    Action(
+        SendValueInterp {
+            field_amount: JsonStringAccumulate::<64>,
+            field_from_address: JsonStringAccumulate::<64>,
+            field_to_address: JsonStringAccumulate::<64>,
```

Comment:
Is the reason we cannot continue to use `show_address` filed order?

Reply:
Yes, to ensure prompts get displayed in correct order, irrespective of field order